src/components/Modal/Modal.js

```diff
@@ -107,7 +107,8 @@ export const Modal = ({
   const modalClassName = "modal-wrapper radius-md";
 
   return ReactDom.createPortal(
-    <div role="dialog" aria-modal="true">
+    <div role="dialog" aria-modal="true" aria-labelledby="">
```

Comment:
Is the `h2` ID supposed to be here?
```suggestion
    <div role="dialog" aria-modal="true" aria-labelledby="modal-title">
```

Reply:
updated in last commit